api/src/main/resources/workbench-api.yaml

```diff
@@ -804,16 +804,22 @@ paths:
           description: Workspace deletion request accepted
           schema:
             "$ref": "#/definitions/EmptyResponse"
-  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cdr-selectors":
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/workspaceResources":
     parameters:
       - "$ref": "#/parameters/workspaceNamespace"
       - "$ref": "#/parameters/workspaceId"
-    get:
+    post:
+      parameters:
+        - in: body
+          name: resourcesToFetch
```

Comment:
It's technically `resourceTypesToFetch`, since you're not naming individual resources.